src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -29,9 +29,9 @@
 import java.text.SimpleDateFormat;
 
 public final class QuizTimingPropertiesUtils {
-
+ 
```

Comment:
A bunch of space characters were added to this file. Removing those will make the number of "changed" lines go down and make this PR easier to review. Also, we shouldn't be adding random spaces to empty lines anyway.